lib/screens/organisation_registration/organisation_registration.dart

```diff
@@ -0,0 +1,87 @@
+import 'dart:io';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'widgets/name_field.dart';
+import 'widgets/tagline_field.dart';
+import 'package:image_picker/image_picker.dart';
+class AddOrganisation extends StatefulWidget {
+  @override
+  _AddOrganisationState createState() => _AddOrganisationState();
+}
+
+class _AddOrganisationState extends State<AddOrganisation> {
+  final _formKey = GlobalKey<FormState>();
+  final TextEditingController nameController = TextEditingController();
+  final TextEditingController taglineController = TextEditingController();
+  File _image;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          title: Text("TEESCO",style: TextStyle(color: Colors.black),),
+          backgroundColor: Colors.white,
+          leading: GestureDetector(
```

Comment:
Instead of GestureDetector, use IconButton
applies to all places where the child of a GestureDetector is an Icon

Reply:
changing this to IconButton?